tests/rt_utils.sh

```diff
@@ -451,7 +451,7 @@ EOF
   fi
 
   cat << EOF >> "${ROCOTO_XML}"
-    <cores>${BUILD_CORES}</cores>
+    <nodes>1:ppn=${BUILD_CORES}</nodes>
```

Comment:
Why was this change made? @SamuelTrahanNOAA

Reply:
nodes is preferred with new version of rocoto. some machine still ok with cores though.

Reply:
Ok. Thanks.